src/assets/scripts/client/airport/runway/RunwayModel.js

```diff
@@ -119,6 +119,15 @@ export default class RunwayModel extends BaseModel {
          */
         this.queue = [];
 
+        /**
+         * The flight number of the last aircraft that used the runway for takeoff.
+         *
+         * @property lastDepartedAircraftId
+         * @type {string}
+         * @default null
+         */
+        this.lastDepartedAircraftId = null;
```

Comment:
Looks like this has since been changed to use the aircraft's callsign instead of its id number. We should either:
a. Change this property name to `.lastDepartedAircraftCallsign`, or
b. Change all uses of this property to store and expect to receive an aircraft id number instead of a callsign

In both cases, the docblock here should be updated for specificity, as `the flight number` implies a portion of the callsign (`1134 of AAL1134`), when it is actually something else.

Reply:
done